docroot/themes/custom/mass_theme/templates/block/block--mainnavigation-3.html.twig

```diff
@@ -43,7 +43,15 @@
               class="ma__header__hamburger__menu-button js-header-menu-button">
         <span class="ma__header__hamburger__menu-icon"></span>
```

Comment:
why do we duplicate the main menu twig template in openmass instead of using the component from mayflower?

Reply:
@clairesunstudio As far as I know, Drupal cannot pull some MF templates directly based on where components locate in the Drupal structure.  This causes a necessary of setting up duplicate templates to Mayflower's in Drupal.